Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM busybox:stable-musl as build
+
+FROM scratch
+
+ARG ARCH=amd64
+
+COPY ./circleci-agent-${ARCH} /opt/circleci/circleci-agent
+COPY --from=build /bin/cp /bin/cp
```

Comment:
~~ Not that it matters much here, but each COPY introduces a new layer which can surprising increase the image size. We can reduce this to one layer: https://stackoverflow.com/a/69010574

Reply:
Nice suggestion. Saves 1Mb